### PR TITLE
provider/google: SSL Certificates resource + tests & documentation

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: website/source/docs/providers/google/r/compute_ssl_certificate.html.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: "google"
+page_title: "Google: google_compute_ssl_certificate"
+sidebar_current: "docs-google-compute-ssl-certificate"
+description: |-
+  Creates an SSL Certificate resource necessary for HTTPS load balancing in GCE.
+---
+
+# google\_compute\_address
```
Comment (sparkprime) [inline website/source/docs/providers/google/r/compute_ssl_certificate.html.markdown]: paste error, and first line of text below

File: website/source/docs/providers/google/r/compute_ssl_certificate.html.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A unique name for the resource, required by GCE.
+    Changing this forces a new resource to be created.
+* `description` - (Optional) The Region in which the created address should reside.
```
Comment (sparkprime) [inline website/source/docs/providers/google/r/compute_ssl_certificate.html.markdown]: repeated and talks about region

File: builtin/providers/google/resource_compute_ssl_certificate.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+	if v, ok := d.GetOk("description"); ok {
+		cert.Description = v.(string)
+	}
+
+	op, err := config.clientCompute.SslCertificates.Insert(
+		config.Project, cert).Do()
+
+	if err != nil {
+		return fmt.Errorf("Error creating certificate: %s", err)
```
Comment (sparkprime) [inline builtin/providers/google/resource_compute_ssl_certificate.go]: ssl certificate

File: builtin/providers/google/resource_compute_ssl_certificate.go [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+		config.Project, d.Id()).Do()
+	if err != nil {
+		if gerr, ok := err.(*googleapi.Error); ok && gerr.Code == 404 {
+			// The resource doesn't exist anymore
+			d.SetId("")
+
+			return nil
+		}
+
+		return fmt.Errorf("Error reading certificate: %s", err)
```
Comment (sparkprime) [inline builtin/providers/google/resource_compute_ssl_certificate.go]: ssl certificate

File: builtin/providers/google/resource_compute_ssl_certificate.go [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+	return nil
+}
+
+func resourceComputeSslCertificateDelete(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	op, err := config.clientCompute.SslCertificates.Delete(
+		config.Project, d.Id()).Do()
+	if err != nil {
+		return fmt.Errorf("Error deleting certificate: %s", err)
```
Comment (sparkprime) [inline builtin/providers/google/resource_compute_ssl_certificate.go]: ssl certificate

**Comment by phinze** [comment]
Hiya @lwander - the acceptance tests here fail for anybody without a keypair in `~/cert`. Do you think you could generate a self-signed cert/key and drop them into `test-fixtures` and change the references to use those?

**Comment by lwander** [comment]
@phinze - sounds good

**Pull request merged**